packages/swapper/src/swappers/LifiSwapper/endpoints.ts

```diff
@@ -98,7 +98,13 @@ export const lifiApi: SwapperApi = {
     const { to, value, data, gasLimit } = transactionRequest
 
     // checking values individually to keep type checker happy
-    if (to === undefined || value === undefined || data === undefined || gasLimit === undefined) {
+    if (
+      to === undefined ||
+      from === undefined ||
```

Comment:
from is defined as `string` in the args. though feels like we can add the `isExecutableTradeQuote` here as well

Reply:
🧠  4c316cb35f